README.md

```diff
@@ -22,3 +159,8 @@ See [Issues](https://github.com/App-vNext/Simmy/issues) for latest discussions a
 ## Credits
 
 Simmy was [the brainchild of](https://github.com/App-vNext/Polly/issues/499) [@mebjas](https://github.com/mebjas) and [@reisenberger](https://github.com/reisenberger). The major part of the implementation was by [@vany0114](https://github.com/vany0114) and [@mebjas](https://github.com/mebjas), with contributions also from [@reisenberger](https://github.com/reisenberger) of the Polly team.
+
+TODO: 
+* examples with other overloads
+* async examples
+* descriptions
```

Comment:
I wouldn't worry about covering every overload, or async examples, too much.  I think users will probably only use Simmy if they are already using (and understand) Polly.  Maybe we should add a note to explain that?  Eg:

>Simmy integrates tightly with the Polly resilience project.  For more general background on policies, PolicyWrap, synchronous vs asynchronous executions and other matters, see also the [main Polly repo](https://github.com/app-vnext/polly).

(We can say more or less here about these issues - I don't mind. I'm just trying to suggest ways to reduce the amount of work, so that we can get the Simmy readme ready for a launch! 😀 )